apps/src/sites/code.org/pages/public/learn/local.js

```diff
@@ -27,14 +27,32 @@ $(function() {
   });
 
   setFacetDefaults();
-
-  $('#location')
-    .geocomplete()
-    .bind('geocode:result', function(event, result) {
-      var loc = result.geometry.location;
-      map_loc = loc.lat() + ',' + loc.lng();
+  if (window.location.search.includes('mapbox')) {
+    $('#location').hide();
+    $('#mapboxgeocoder').show();
+    var geocoder = new MapboxGeocoder({
+      accessToken: mapboxgl.accessToken,
+      mapboxgl: mapboxgl,
+      marker: false,
+      types: 'country,region,district,postcode,locality,place'
+    });
+    geocoder.on('result', function(result) {
+      var loc = result.result.geometry.coordinates;
+      map_loc = loc[1] + ',' + loc[0];
       submitForm();
     });
+    geocoder.addTo('#mapboxgeocoder');
+  } else {
```

Comment:
Up to you whether to do this in this PR or a follow up, but we should start showing the Mapbox stuff by default 😄

Reply:
oh right that is tomorrow hahaha, I'll take out the google maps stuff now